VideoLocker/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -228,6 +233,18 @@ public String getNewRelicKey() {
     }
 
     /**
+     * End to end test configurations
+     */
+    public class EndToEndConfig {
+        private String DEFAULT_EMAIL_TEMPLATE = "test-{unique_id}@example.com";
```

Comment:
Because it's logically separated from it's parent class, and doesn't need to include a reference to it. In fact, it's not possible for it to be a valid non-static inner class, since it's instantiated by Gson view reflection. It would usually throw an exception, but Gson uses an unsafe technique to have it instantiated nonetheless, which means that you will get a strange error if you actually try to access any property of it's outerclass.